src/error/exporterrors.cpp

```diff
@@ -524,6 +524,7 @@ static ostream & printPrivate (ostream & os, parse_t & p)
 	   << endl
 	   << "#include <elektra/types.h>" << endl
 	   << "#include <elektra/error.h>" << endl
+	   << "#include <kdbprivate.h>" << endl
```

Comment:
Why is this needed? It would be good if we can avoid it. If it is only about the enum, we better move the enum within here.

Reply:
It's needed for `ElektraErrorSeverity`, which is used in the generated function `elektraErrorLowLevel`. The function is used to create the `ElektraError` struct for low-level errors.

I don't think it is a problem to include `kdbprivate.h` here. The generated file is `elektra/errorsprivate.h` and is not intended for external use. It is not even included in any CMake install targets. It is not referenced in any header files, so it basically doesn't exist outside of compiling Elektra.